knotx-server/src/main/java/io/knotx/server/KnotxServerVerticle.java

```diff
@@ -49,7 +51,9 @@ public void start(Future<Void> fut) throws IOException, URISyntaxException {
 
     configuration.getDefaultFlow().getEngineRouting().forEach((key, value) -> {
       if (key == HttpMethod.POST) {
-        router.route().method(key).handler(BodyHandler.create());
+        BodyHandler bodyHandler = StringUtils.isNoneEmpty(configuration.getFileUploadDirectory()) ?
```

Comment:
Should be "not" not "none" and notBlank will be better. It is possible to send blank value and it cause troubles in vert.x